### PR TITLE
Upgrade laminas-coding-standard to 2.3

### PR DESCRIPTION
**Pull request by alexraputa**
Closes #26

|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### CODE REVIEW

File: src/Listener/LocatorRegistrationListener.php [hunk 1]
```diff
@@ -14,33 +10,25 @@
 use Laminas\ModuleManager\Feature\LocatorRegisteredInterface;
 use Laminas\ModuleManager\ModuleEvent;
 use Laminas\ModuleManager\ModuleManager;
+use Laminas\Mvc\Application;
 use Laminas\Mvc\MvcEvent;
-use Laminas\ServiceManager\ServiceManager;
```
Comment (Xerkus) [inline src/Listener/LocatorRegistrationListener.php:18]: ServiceManager is used in typehint on line 65 and 93

File: src/Listener/ServiceListener.php [hunk 2]
```diff
@@ -184,7 +166,7 @@ public function onLoadModule(ModuleEvent $e)
             // We are keeping track of which modules provided which configuration to which service managers.
             // The actual merging takes place later. Doing it this way will enable us to provide more powerful
             // debugging tools for showing which modules overrode what.
-            $fullname = $e->getModuleName() . '::' . $sm['module_class_method'] . '()';
+            $fullname                                                = $e->getModuleName() . '::' . $sm['module_class_method'] . '()'; /** @codingStandardsIgnoreLine */
```
Comment (Xerkus) [inline src/Listener/ServiceListener.php]: And this is why I really dislike the alignment rule in out coding standard.

File: test/ModuleManagerTest.php [hunk 3]
```diff
@@ -187,8 +180,8 @@ public function testCanLoadSomeObjectModule()
         require_once __DIR__ . '/TestAsset/SubModule/Sub/Module.php';
         $configListener = $this->defaultListeners->getConfigListener();
         $moduleManager  = new ModuleManager([
-            'SomeModule' => new \SomeModule\Module(),
-            'SubModule' => new \SubModule\Sub\Module(),
+            'SomeModule' => new Module(),
+            'SubModule'  => new \SubModule\Sub\Module(),
```
Comment (Xerkus) [inline test/ModuleManagerTest.php]: This should be imported as well

**Review by Xerkus — CHANGES_REQUESTED** [review]
Looks good.
There are just couple minor changes needed.

File: src/Listener/ServiceListener.php (hunk 2 above)
Comment (alexraputa) [reply src/Listener/ServiceListener.php]: Agree.

File: src/Listener/LocatorRegistrationListener.php (hunk 1 above)
Comment (alexraputa) [reply src/Listener/LocatorRegistrationListener.php:18]: Oops )

**Comment by alexraputa** [comment]
@Xerkus, done.

File: test/TestAsset/ModuleAsClass.php [hunk 4]
```diff
@@ -1,10 +1,8 @@
 <?php
 
-/**
- * @see       https://github.com/laminas/laminas-modulemanager for the canonical source repository
- * @copyright https://github.com/laminas/laminas-modulemanager/blob/master/COPYRIGHT.md
- * @license   https://github.com/laminas/laminas-modulemanager/blob/master/LICENSE.md New BSD License
- */
+declare(strict_types=1);
+
+namespace LaminasTest\ModuleManager\TestAsset;
```
Comment (Xerkus) [inline test/TestAsset/ModuleAsClass.php]: I don't see associated change elsewhere. Is this class unused?

Comment (alexraputa) [reply test/TestAsset/ModuleAsClass.php]: Yes.

Comment (Xerkus) [reply test/TestAsset/ModuleAsClass.php]: Then drop it entirely.

Comment (alexraputa) [reply test/TestAsset/ModuleAsClass.php]: > Then drop it entirely.

Done.

**Review by Xerkus — APPROVED** [review]

**Comment by Xerkus** [comment]
PHP 8.1 failure is unrelated, ignoring it.

**Comment by Xerkus** [comment]
@alexraputa thank you

**Pull request merged**